aiohttp/helpers.py

```diff
@@ -1063,3 +1071,13 @@ def parse_http_date(date_str: Optional[str]) -> Optional[datetime.datetime]:
             with suppress(ValueError):
                 return datetime.datetime(*timetuple[:6], tzinfo=datetime.timezone.utc)
     return None
+
+
+def verify_ssl_type(
+    ssl: Optional[Union[SSLContext, Literal[False], Fingerprint]]
```

Comment:
It's already a union.
```suggestion
    ssl: Union[SSLContext, Literal[False], Fingerprint, None]
```